README.md

```diff
@@ -318,10 +318,34 @@ Adds new parsers for custom file extensions
 
 Adds a new custom format.
 
-### config.addFormats(formatArray)
+### config.addFormats(formatsObject)
 
 Adds new custom formats.
 
+```javascript
+convict.addFormats({
+  {
+    name: 'float-percent',
```

Comment:
I ran into this as well. You're right about the argument being an object, but the key for each element is the `name` property.

Something like: `{ 'float-percent': { validate, coerce }, 'hex-string': { validate, coerce } }`